shared/actions/chat2/index.tsx

```diff
@@ -112,18 +112,16 @@ function* inboxRefresh(
     )
   }
 
-  yield* Saga.callRPCs(
```

Comment:
i simplified / removed some utils from Saga. This was to help us with a flow issue but isn't necessary